### PR TITLE
Fix svggepard build

### PR DESCRIPTION
**Pull request by szledan**
Signed-off-by: Szilard Ledan <szledan@gmail.com>

### CODE REVIEW

File: apps/svggepard/CMakeLists.txt [hunk 1]
```diff
@@ -7,4 +7,5 @@ add_executable(svggepard ${SOURCES})
 find_package(X11)
 find_package(PNG)
 
+target_include_directories(svggepard PUBLIC ${INCLUDE_OUTPUT_DIR})
```
Comment (dbatyai) [inline apps/svggepard/CMakeLists.txt]: Do we need this to be public?

Comment (szledan) [reply apps/svggepard/CMakeLists.txt]: Yes, you are right. The `PRIVATE` is justifiable.

Comment (kkristof) [reply apps/svggepard/CMakeLists.txt]: I think you might changed the wrong target include directory

Comment (dbatyai) [reply apps/svggepard/CMakeLists.txt]: Both of these should be private I believe, so it's not completely wrong.

Comment (dbatyai) [reply apps/svggepard/CMakeLists.txt]: Both of these should be private I believe, so it's not completely wrong.

**Pull request merged**